package.json

```diff
@@ -13,6 +13,7 @@
     "build:node": "tsc -p .",
     "build:types": "dts-bundle-generator -o ./types/index.d.ts ./src/index.ts --project ./tsconfig.types.json --no-check",
     "clean": "rimraf ./build && rimraf ./types",
+    "prettier": "prettier '**/*.ts' --check",
```

Comment:
in line 8 the ci script is also linting. So I guess your script is not separating the concerns of the tests.

Should be changed to:
"ci": "npm run build && npm run test",

Reply:
fixed